documentation/faq/80-integrations.md

```diff
@@ -106,3 +106,29 @@ onMount(() => {
 ### Does it work with Yarn 2?
 
 Sort of. The Plug'n'Play feature, aka 'pnp', is broken (it deviates from the Node module resolution algorithm, and [doesn't yet work with native JavaScript modules](https://github.com/yarnpkg/berry/issues/638) which SvelteKit — along with an [increasing number of packages](https://blog.sindresorhus.com/get-ready-for-esm-aa53530b3f77) — uses). You can use `nodeLinker: 'node-modules'` in your [`.yarnrc.yml`](https://yarnpkg.com/configuration/yarnrc#nodeLinker) file to disable pnp, but it's probably easier to just use npm or [pnpm](https://pnpm.io/), which is similarly fast and efficient but without the compatibility headaches.
+
+### Yarn 3 Berry Example
+
+Currently ESM Support within the latest yarn (version 3) is considered experimental.  
+The below seems to work although you're results may vary.
```

Comment:
Changes now made, I do recognise that there may be unforeseen problems I've not picked up on and that its unsupported etc. But I just noticed it worked as is and thought it might be useful to mention the config change.

Personally I've used pnpm before but then moved to yarn berry with enableGlobalCache since it seems like a cleaner option moving forward without all the filesystem links.

Reply:
Ok I just did a bit more testing on this
It seems to work the first time you run "yarn dev", but then fails on attempts afterwards with a "p.replace is not a function" error. Until you delete the .svelte-kit directory